bokehjs/src/coffee/models/glyphs/annular_wedge.coffee

```diff
@@ -31,20 +34,20 @@ class AnnularWedgeView extends Glyph.View
 
       ctx.moveTo(souter_radius[i], 0)
       ctx.beginPath()
-      ctx.arc(0, 0, souter_radius[i], 0, angle[i], direction[i])
+      ctx.arc(0, 0, souter_radius[i], 0, angle[i], direction)
       ctx.rotate(@angle[i])
       ctx.lineTo(sinner_radius[i], 0)
-      ctx.arc(0, 0, sinner_radius[i], 0, -angle[i], not direction[i])
+      ctx.arc(0, 0, sinner_radius[i], 0, -angle[i], not direction)
       ctx.closePath()
 
       ctx.rotate(-angle[i]-start_angle[i])
       ctx.translate(-sx[i], -sy[i])
 
-      if @visuals.fill.do_fill
+      if @visuals.fill.do
```

Comment:
Surprising this works. `do` is a keyword in CoffeeScript. I suggest renaming this to something else, e.g.  `doit`.